met-api/tests/unit/api/test_language.py

```diff
@@ -0,0 +1,108 @@
+# Copyright © 2019 Province of British Columbia
```

Comment:
We checked with Saravan and there are no copyrights for this app. Could you please remove this section?